### ISSUE TITLE
sgds-bottom-navigation background color and text on-hover color lack contrast

### ISSUE THREAD

**Issue by Sylphias** [GovTechSG/sgds#58]
In SGDS css, the declaration for the color for the bottom navigation is blue-ish (#4675d3)
```
.sgds-bottom-navigation a {
       color: white;
       background-color: #4675d3;
       display: block;
       width: 50%;
       padding: 1rem 5rem;
}
```
Text highlights in this same navigator is also blue-ish (#2c5bba)
```
.sgds-bottom-navigation a:hover {
     background-color: #2c5bba;
}
```

Suggestion to change color of a:hover to something with better contrast
